tests/e2e/pageobjects/openshift/OcpMainPage.ts

```diff
@@ -84,4 +88,22 @@ export class OcpMainPage {
         await this.driverHelper.enterValue(OcpMainPage.PROJECT_FILTER_INPUT_LOCATOR, projectName);
         await this.driverHelper.waitAndClick(OcpMainPage.getProjectDropdownItemLocator(projectName));
     }
+
+    private async selectRole(role: string): Promise<void> {
+        Logger.debug(`${this.constructor.name}.${this.selectRole.name} - ${role}`);
+
+        await this.driverHelper.waitAndClick(OcpMainPage.getRoleLocator(role));
+    }
+
+    private async tryToSkipWebTour(): Promise<void> {
+        Logger.debug(`${this.constructor.name}.${this.tryToSkipWebTour.name}`);
+
+        if (await this.driverHelper.isVisible(OcpMainPage.SKIP_TOUR_BUTTON_LOCATOR)) {
+            await this.driverHelper.waitAndClick(OcpMainPage.SKIP_TOUR_BUTTON_LOCATOR);
+
+            Logger.debug(`${this.constructor.name}.${this.tryToSkipWebTour.name} - Welcome tour modal dialog was located and skipped`);
```

Comment:
IMHO, it would improve code readability if get it to from utility method, like following:
```
function getCallerInfo(): string {
  const error = new Error();

  // Use regular expressions to parse the stack trace
  const stackTrace = error.stack || '';
  const classNameMatch = stackTrace.match(/at\s+(\S+)\./);
  const className = classNameMatch ? classNameMatch[1] : undefined;

  let methodNameMatch;
  if (className) {
    const methodNameRegex = new RegExp(`at ${className}\\.(\\S+)`);
    methodNameMatch = stackTrace.match(methodNameRegex);
  }

  const methodName = methodNameMatch ? methodNameMatch[1] : undefined;

  return `${className}.${methodName}`
}
```

Reply:
Agree, but it is better to improve our Logger class, but this is out of scope here